### PR TITLE
chore: add enableSwapFee to TriggerInputShape for swap-deposit

### PR DESCRIPTION
**Pull request by silasbw**

### CODE REVIEW

File: src/types/shortcuts.ts [hunk 1]
```diff
@@ -31,12 +31,22 @@ export const tokenAmountWithMetadata = z.object({
   isNative: z.boolean(),
 })
 
+export const ZodEnableSwapFee = z
```
Comment (satish-ravi) [inline src/types/shortcuts.ts:34]: this is a post request, can we just use zod booleans?

File: src/apps/allbridge/shortcuts.ts [hunk 2]
```diff
@@ -192,6 +193,7 @@ const hook: ShortcutsHook = {
         triggerInputShape: {
           swapFromToken: tokenAmountWithMetadata,
           // set via shortcutTriggerArgs, the deposit token and position addresses
+          enableSwapFee: ZodEnableSwapFee,
```
Comment (satish-ravi) [inline src/apps/allbridge/shortcuts.ts:196]: can we move this above the comment? This has to come from the wallet

File: src/types/shortcuts.ts (hunk 1 above)
Comment (silasbw) [reply src/types/shortcuts.ts:34]: I don't think so:

> Note — Boolean coercion with z.coerce.boolean() may not work how you expect. Any [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) value is coerced to true, and any [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) value is coerced to false.

**Comment by silasbw** [comment]
this doesn't the behavior of anything yet @MuckT  --  I'm mostly trying to propose the parameter name and values so we can get the wallet change done.

File: src/types/shortcuts.ts (hunk 1 above)
Comment (silasbw) [reply src/types/shortcuts.ts:34]: doh, yes we can.

**Review by MuckT — APPROVED** [review]

**Pull request merged**